prow/kube/prowjob.go

```diff
@@ -152,6 +175,8 @@ type Refs struct {
 	BaseSHA string `json:"base_sha,omitempty"`
 
 	Pulls []Pull `json:"pulls,omitempty"`
+
+	PathAlias string `json:"path_alias,omitempty"`
```

Comment:
Would expect URI overrides and pull secret configs to follow here, to ensure that this remains a declarative config that can explain what to pull and how